### PR TITLE
Rename library enum

### PR DESCRIPTION
**Pull request by nbierdeman**
### Description

We have decided on a new name for tracking library usage, thus this PR. Alphabetization is in a separate commit, so it's easier to see the core changes. The original PR is [here](https://github.com/paypal/paypal-sdk-constants/pull/140).

### JIRA

DTPPCPSDK-1482

### CODE REVIEW

**Review by jshawl — APPROVED** [review]
:shipit: (sanity checking - does the existing constant need to remain in-place to prevent a failed build in paypal-checkout-components)? e.g.: add a new variable `JS_SDK_LIBRARIES` then change checkout components, then come back and delete? just looking out - I think you'd be able to make the checkout components change before the next publish

**Comment by nbierdeman** [comment]
@jshawl, `SDK_INTEGRATION_SOURCES` is only being used [here](https://github.com/paypal/react-paypal-js/pull/420), which I should probably mark as a draft anyway, since I still need to enforce a library value there.

**Review by wsbrunson — APPROVED** [review]

File: src/params.js [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
-  AMOUNT: ("data-amount": "data-amount"),
-  CLIENT_METADATA_ID: ("data-client-metadata-id": "data-client-metadata-id"),
-  PAGE_TYPE: ("data-page-type": "data-page-type"),
+  SDK_TOKEN: ("data-sdk-client-token": "data-sdk-client-token"),
+  STAGE_HOST: ("data-stage-host": "data-stage-host"),
   USER_EXPERIENCE_FLOW:
     ("data-user-experience-flow": "data-user-experience-flow"),
-  POPUPS_DISABLED: ("data-popups-disabled": "data-popups-disabled"),
-  SDK_TOKEN: ("data-sdk-client-token": "data-sdk-client-token"),
+  USER_ID_TOKEN: ("data-user-id-token": "data-user-id-token"),
```
Comment (jshawl) [inline src/params.js:31]: 🔤 💯 🥳

**Review by jshawl — APPROVED** [review]
🚀

**Pull request merged**